docs/victoriametrics/data-ingestion/OpenShift.md

```diff
@@ -0,0 +1,95 @@
+---
+title: OpenShift
+weight: 7
+menu:
+  docs:
+    identifier: "openshift"
+    parent: "data-ingestion"
+    weight: 7
+tags:
+  - metrics
+aliases:
+  - /data-ingestion/openshift.html
+  - /data-ingestion/OpenShift.html
+  - /data-ingestion/openshift/index.html
+  - /data-ingestion/openshift/
+---
+
+## OpenShift Container Platform (OCP)
+
+OpenShift uses Prometheus as a core monitoring solution. It cannot be replaced without violating the OpenShift support policy. However, OpenShift can be configured to use VictoriaMetrics as a remote write target.
+
+According to [remote write configuration in the OpenShift documentation](https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/monitoring/configuring-core-platform-monitoring#configuring-remote-write-storage_configuring-metrics) the following manifest needs to be applied to send platform metrics to VictoriaMetrics:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "http://vminsert-hub.victoriametrics.svc.cluster.local:8480/insert/1/prometheus/api/v1/write"
+        writeRelabelConfigs:
+        - regex: 'prometheus_replica'
+          action: labeldrop
+```
+This instructs Prometheus to copy collected metrics to specified VictoriaMetrics ingester.
+
+Note, that OpenShift uses two Prometheus replicas for HA configuration. Metric series are identified by `prometheus_replica` label. This is excessive on VictoriaMetrics side, so this label needs to be dropped to avoid inserting duplicate metrics.
```

Comment:
```suggestion
Note, that OpenShift uses two Prometheus replicas for HA configuration. Each replica adds `prometheus_replica` label to the forwarded metrics. This is excessive on VictoriaMetrics side, so this label needs to be dropped to avoid inserting duplicate metrics.
```